manifests/r/RescueTime/DesktopApp/2.16.8.2/RescueTime.DesktopApp.locale.en-US.yaml

```
@@ -0,0 +1,28 @@
+# Created using wingetcreate 1.0.3.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: RescueTime.DesktopApp
+PackageVersion: 2.16.8.2
+PackageLocale: en-US
+Publisher: RescueTime
+PublisherUrl: https://www.rescuetime.com/
+PublisherSupportUrl: https://help.rescuetime.com/category/200-apps
+PackageName: RescueTime
+PackageUrl: https://www.rescuetime.com/get_rescuetime
+License: Proprietary
+ShortDescription: Desktop client for RescueTime time tracking service (rescuetime.com)
+Description: Desktop client for RescueTime time tracking service (rescuetime.com). Automatically track and get insights on how you spend your time on apps and websites. No manual time entry, finicky timers, or start/stop buttons required. Free version available; upgrade required for full feature set.
+Moniker: RescueTime
+Tags:
+- rescuetime
+- time tracking
+- productivity
+- time management
+- digital wellness
+- Rescue Time
+- rescuetime.com

```

Some of these tags are redundant with the package name, and will never be used; Standard format is to replace spaces with dashes
```suggestion
Tags:
- time-tracking
- productivity
- time-management
- digital-wellness
```